### ISSUE TITLE
Haven't pulled any request data from Socrata since 8/14

### ISSUE THREAD

**Issue by nichhk** [hackforla/311-data#1327]
### Overview
I checked our dev site today, and noticed that no requests showed up. I tried pulling data for an earlier time range, and found that our most recent data pull is from 8/14. The prefect logs on AWS reflect this. E.g., see the logs from today: 
```
  | 2022-08-23T01:03:34.550-07:00 | [2022-08-23 08:03:34+0000] INFO - prefect \| "Loading Socrata data to Postgres" loaded [0] records, updated [0] records, and finished with message "All reference tasks succeeded."
```

### Action Items
- [ ] Manually check the Socrata API to see if any data has been published in the past week

### Resources/Instructions
https://github.com/hackforla/311-data/blob/dev/docs/data_loading.md

**Comment by nichhk** [comment]
I tried manually reading the request data directly from the API using this URL: https://data.lacity.org/resource/i5ke-k6by.json?$where=updateddate%20%3E%202022-08-13T22:39:18.000. I figured out how to form this URL by looking at the Socrata [docs](https://dev.socrata.com/foundry/data.lacity.org/i5ke-k6by) for the 2022 LA 311 dataset.

This gets requests that have been updated since 8/13/22. The API responds with:
```
{
  "error" : true,
  "message" : "Internal error",
  "status" : 500
}
```

Next steps--try running Prefect locally and see if it can retrieve more recent data.
